src/config.ts

```diff
@@ -0,0 +1,35 @@
+import { Logger } from '@opentelemetry/api';
+import * as http from 'http';
+import * as https from 'https';
+
+export interface ExporterConfig {
+  // Buffer
+  maxStorageSize: number; // number of envelopes
+
+  // Exporter
+  instrumentationKey?: string;
+  connectionString?: string;
+
+  // Channel
+  maxBatchSizeBeforeSend: number; // number of envelopes
+  batchSendIntervalMs: number;
+  batchSendRetryIntervalMs: number;
+  logger?: Logger;
+
+  // Sender
+  maxConsecutiveFailuresBeforeWarning: number;
+  endpointUrl: string;
+  proxyHttpsUrl?: string;
+  proxyHttpUrl?: string;
+  httpAgent?: http.Agent;
+  httpsAgent?: https.Agent;
+}
+
+export const DEFAULT_EXPORTER_CONFIG: ExporterConfig = {
+  maxStorageSize: 2_000,
```

Comment:
Is this number of entries or `bytes`? I believe we use `byte` amount for all other SDKs.

Reply:
For this one it was entries, but I am removing this config altogether